### ISSUE TITLE
1.4.1 doesn't work on CentOS6

### ISSUE THREAD

**Issue by slandelle** [hyperxpro/Brotli4j#21]
```
java.lang.UnsatisfiedLinkError: /tmp/com_aayushatharva_brotli4j_1036749389702/libbrotli.so: /usr/lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /tmp/com_aayushatharva_brotli4j_1036749389702/libbrotli.so)
```

We're investigating.

FYI, it works fine on latest Amazon Linux (1.3.2 didn't) 🤷

**Comment by hyperxpro** [comment]
We compiled this on CentOS 6 but it's not running on CentOS 6 itself. 😂

**Comment by slandelle** [comment]
Because we compiled on CentOS 6 with the c++ compiler installed but somehow one piece is missing when running the binary on CentOS 6 when the c++ compiler is not there.
What's weird is that it works on Amazon Linux 2?!

**Comment by hyperxpro** [comment]
Fixed by #23
